RedHat-UX/red-hat-design-system
# docs(support): adding Support page

## What I did

1. Added [Get support page](https://deploy-preview-1642--red-hat-design-system.netlify.app/support/)
2. Added `margin-block-end` styles to `h2`, `h3`, and `h4` element in `backpage.css` to make the vertical rhythm nicer, since without it, the headings were flush to the tiles.


## Testing Instructions

1. Compare to [Figma mockup](https://www.figma.com/design/s2jasqNAZ2LljpbbmfEPog/Support?node-id=0-1&m=dev)
2. Double check other backpages to make sure the vertical rhythm looks nice, and that the heading margins didn't get messed up.



Closes: #1443
